core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -153,11 +157,14 @@ public LockResource lockShared() throws InterruptedException {
    */
   public LockResource lockExclusive(StateLockOptions lockOptions)
       throws TimeoutException, InterruptedException {
+    LOG.debug("Thread-%d entered lockExclusive().", Thread.currentThread().getId());
     // Run the grace cycle.
     StateLockOptions.GraceMode graceMode = lockOptions.getGraceMode();
     boolean lockAcquired = false;
     long deadlineMs = System.currentTimeMillis() + lockOptions.getGraceCycleTimeoutMs();
     while (System.currentTimeMillis() < deadlineMs) {
+      LOG.debug("Thread-%d entered grace-cycle of try-sleep: %ms-%ms",
```

Comment:
I don't think this syntax is correct. It doesn't have enough arguments, and is "%ms" correct?